Sources/StreamVideoSwiftUI/CallViewModel.swift

```diff
@@ -177,21 +174,8 @@ open class CallViewModel: ObservableObject {
         )
     }
     
-    public var participants: [CallParticipant] {
-        callParticipants
-            .filter {
-                if participantsLayout == .grid,
-                   callParticipants.count <= 3,
-                   (call?.state.screenSharingSession == nil || call?.state.isCurrentUserScreensharing == true) {
```

Comment:
Why do we remove this condition? If I remember correctly, this prevented mirroring when the current user is screensharing. There are several cases here that we should check.

Reply:
Can you elaborate more about what this purposed for? What do you mean by mirroring? I removed it to maintain consistency with state's participants and because screensharing is being handled from sorting. What shall i be looking for here?

Reply:
This one is now covered by unit tests